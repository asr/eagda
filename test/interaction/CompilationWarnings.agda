-- See also test/Succeed/InlineCompiled.agda
module _ where

id : {A : Set} → A → A
id x = x
{-# INLINE id #-}

-- this is pointless and should generate a warning in the info buffer
{-# COMPILED id \ _ x -> x #-}

