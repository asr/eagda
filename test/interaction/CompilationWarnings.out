(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking CompilationWarnings (CompilationWarnings.agda). " t)
(agda2-info-action "*Type-checking*" "Finished CompilationWarnings. " t)
(agda2-info-action "*Type-checking*" "Compiling Agda.Primitive in agda-default-include-path/Agda/Primitive.agdai to MAlonzo/Code/Agda/Primitive.hs " t)
(agda2-info-action "*Type-checking*" "Compiling CompilationWarnings in CompilationWarnings.agdai to MAlonzo/Code/CompilationWarnings.hs " t)
(agda2-info-action "*Type-checking*" "Calling: ghc -O -Werror -i MAlonzo/Code/CompilationWarnings.hs --make -fwarn-incomplete-patterns -fno-warn-overlapping-patterns " t)
(agda2-info-action "*Type-checking*" "[1 of 2] Compiling MAlonzo.RTE ( MAlonzo/RTE.hs, MAlonzo/RTE.o ) " t)
(agda2-info-action "*Type-checking*" "[2 of 2] Compiling MAlonzo.Code.CompilationWarnings ( MAlonzo/Code/CompilationWarnings.hs, MAlonzo/Code/CompilationWarnings.o ) " t)
(agda2-status-action "Checked")
(agda2-info-action "*Compilation result*" "The module was successfully compiled. ———— Warnings —————————————————————————————————————————————— CompilationWarnings.agda:9,1-36 It is pointless for INLINE'd function id to have a separate Haskell definition " nil)
((last . 1) . (agda2-goals-action '()))
